packages/chat-component-bindings/src/messageThreadSelector.ts

```diff
@@ -237,7 +237,7 @@ const convertToUiChatMessage = (
   isLargeGroup: boolean
 ): ChatMessage => {
   const messageSenderId = message.sender !== undefined ? toFlatCommunicationIdentifier(message.sender) : userId;
-  /* @conditional-compile-remove(file-sharing) @conditional-compile-remove(teams-inline-images-and-file-sharing) */
+  /* @conditional-compile-remove(file-sharing) */
   const { /* @conditional-compile-remove(file-sharing) */ files } = extractAttachmentsMetadata(message);
```

Comment:
nit: in this case since we are cc-ing out the whole const no need for the `/* @conditional-compile-remove(file-sharing) */`